### ISSUE TITLE
Nuxt app not starting in Safari (Invalid regular expression: invalid group specifier name)

### ISSUE THREAD

**Issue by stefandevo** [ServiceStack/Issues#773]
I am not sure since what version of @servicestack/client but at since 1.0.31 and later a nuxt application using JsonServiceClient is blocked upon startup on Safari -> SyntaxError: Invalid regular expression: invalid group specifier name

Simple project setup:
- follow https://nuxtjs.org/docs/get-started/installation to setup base project
- add `import { JsonServiceClient } from "@servicestack/client"` to the tutorial.vue file, and add a button that calls the `onClick' method:
```
  methods: {
    onClick: function() {
      let json = new JsonServiceClient();
    }
  }
```

Then use the dev npm script on Safari.

The browser opens, shows the spinning wheel, but hangs.
The console shows:

<img width="544" alt="image" src="https://user-images.githubusercontent.com/1656051/155234611-9d510049-ec3b-4e4e-b208-e4ae69815e90.png">

Can be found here:
https://github.com/stefandevo/ssclient_issue

Safari Mac Version 15.2 (17612.3.6.1.6)

**Comment by Layoric** [comment]
Confirmed issue related to use of lookbehind in [the following commit](https://github.com/ServiceStack/servicestack-client/commit/aa086c5a66ff37018061338c0a22aae9a379ddcd#diff-3d2b59189eeedc2d428ddd632e97658fe310f587f7cb63b01f9b98ffc11c0197R1413).

Safari [doesn't yet support this](https://caniuse.com/js-regexp-lookbehind) so an alternative will need to be used.

**Comment by Layoric** [comment]
The look behind regex has been replaced in [this commit](https://github.com/ServiceStack/servicestack-client/commit/363c6d7a6640f94c2b446787dab360b93d8e39fd).

This change is available from v1.1.19 that’s now available on npm.

Thanks for reporting @stefandevo !
